app/views/exchanges/hbx_profiles/_pvp_eligibility_form.html.slim

```diff
@@ -32,7 +32,7 @@
       .modal-header
         h4.modal-title#pvpConfirmModalLabel= @product.title
       .modal-body
-        = l10n('hbx_profiles.pvp_confirmation_question')
+        = l10n('hbx_profiles.pvp_confirmation_question', year: params[:year])
```

Comment:
this key doesn't seem to take a param currently in the l10n mappings yml.

Reply:
Also, I think this PR is missing the updates to this l10n key's value that are shown in the mock. (change to: "Select Confirm to mark the product for the calendar year <YEAR> ... "

Reply:
Oops, you right I forgot to commit this file. Thank you

Reply:
That explains it 😄 !